docs/tofu.logging.recipes.zio.md

```diff
@@ -0,0 +1,168 @@
+---
+id: tofu.logging.recipes.zio
+title: ZIO Logging
+---
+
+## ZIO logging
+To use logging functional adapted for ZIO users, first add the following dependency:
+
+```sbt
+libraryDependencies += "tf.tofu" %% "tofu-logging-zio" % "<latest version in the badge in README>"
+```
+Then import the package:
+```scala
+import tofu.logging.zlogs._
+```
+This contains some useful stuff:
+
+##### `tofu.logging.Logging` type aliases
+These services do logs.
+- `ULogging` — is a type alias for `Logging[UIO]`. Logging methods have no environment like 
+`def info(message: String, values: LoggedValue*): UIO[Unit]`
+- `ZLogging[R]` — is a type alias for `Logging[URIO[R, *]]`. Logging methods require a ZIO environment `R`:
```

Comment:
It's hard to tell why we would need two different `Logging`s. I guess something like "ZLogging is for context logging, ULogging is for everything else" would be enough.

Reply:
What do you mean by "context logging"? `plainLoggingWithContext` is context logging, but encapsulates the context inside, so this is `ULogging`

Reply:
Ah, I see now.